static/js/create_edit.js

```diff
@@ -44,15 +43,10 @@ function txtUpdateOnChange() {
   let shortLink = document.getElementById("shortlink");
   shortLink &&
     shortLink.addEventListener("keyup", function () {
-      let printout = document.getElementById("shortlinked");
+      let printout = document.getElementById("fueledEndPoint");
       var x = shortLink.value;
```

Comment:
je     var x = shortLink.value opravdu potřeba?
jednak, naming x je nicneříkající, pokud není ta definice té proměnné přímo nad tím, tak nikdy nepochopím co to je

proč ne jen     printout.innerHTML = "www.fueled.by/" + shortLink.value ??